new_component_todo-item.md

```diff
@@ -20,25 +22,25 @@ Use the component in the template of `appComponent` - inside the `<li>` element:
 </ul>
 ```
 
-Check out the result in the browser. What do you see? Why?
+Confira o resultado no navegador. O que você  ver? Por que?
 
 ## @Input()
-We want to display the title of each item from within the `todo-item` component. We need to pass the title of the current item in the loop (or the whole item) to the `todo-item` component. 
+Nós queremos exibir o titulo de cada item dentro do componente `todo-item`. Precisamos passar o titulo atual do item  no loop(ou o item inteiro) para o o componente `todo-item`. 
 
-Again, Angular makes it really easy for us, by providing us the `Input` decorator.
+Novamente, Angular faz com que seja muito fácil para nós, fornecendo-nos o decoratore `Input`.
 
-Inside the newly generated `itemComponent` class (itemComponent) add the line:
+Dentro da classe recém criada `itemComponent` (itemComponent) adicione a linha:
 ```ts
 @Input() itemTitle: string;
 ```
-It tells the component to expect an input of type string and to assign it to the class member called `itemTitle`. Make sure that `Input` is added to the import statement in the first line in the file. Now we can use it inside the `itemComponent` template:
+Ele diz ao componente que espera uma entrada do tipo string e atribue ao membro da classe chamada `itemTitle`. Certifique-se  de que o `Input` iestá adicionado na declaração de importação na primeira linha do arquivo.Agora, podemos usá-lo  dentro do template `itemComponent`:
```

Comment:
Please, fix the phrase "Certifique-se  de que o `Input`" ...

Reply:
Thanks, corrected #063c991.